crates/sb_core/js/http.js

```diff
@@ -115,12 +122,15 @@ function serve(args1, args2) {
 
 		try {
 			for await (const requestEvent of currentHttpConn) {
+				ACTIVE_REQUESTS++;
```

Comment:
what's the purpose of tracking `ACTIVER_REQUESTS`? Where is it used?

Reply:
It was used for debugging, but it seems that I didn't clear it.

I'll open another PR to erase this. (Several branches I'm working on directly depend on this PR's commits, so if I modify this PR, I will have to perform an unnecessary force push.)